docs/graphx-programming-guide.md

```diff
@@ -709,7 +709,8 @@ messages remaining.
 
 The following is the type signature of the [Pregel operator][GraphOps.pregel] as well as a *sketch*
 of its implementation (note: to avoid stackOverflowError due to long lineage chains, pregel support periodcally
-checkpoint graph and messages by setting "spark.graphx.pregel.checkpointInterval"):
+checkpoint graph and messages by setting "spark.graphx.pregel.checkpointInterval" to a positive number,
+say 10. And set checkpoint directory as well using SparkContext.setCheckpointDir(directory: String)):
```

Comment:
I reference the value in LDA and other ml algorithms in spark, by default their checkpointInterval is set to 10.